crates/aptos/src/move_tool/mod.rs

```diff
@@ -165,10 +169,10 @@ pub fn init_move_dir(
         Dependency {
             local: None,
             git: Some("https://github.com/aptos-labs/aptos-core.git".to_string()),
```

Comment:
Do we still need git here?

Reply:
Yeah, that was a bug.  Though, i spent time trying to fix this and it was failing still anyways